schema/defs.yaml

```diff
@@ -77,6 +77,11 @@ description: Common schema definitions shared by all themes
         recordId:
           type: string
           description: Refers to the specific record within the dataset that was used.
+        confidence:
+          description: If the source data has a confidence value, especially for ml-derived data, record that here.
+          type: number
+          minValue: 0
+          maxValue: 1
```

Comment:
Interesting — So... 0 is actually a definitive "not here anymore" in the places data, according to the schema docs, which, I think there could be value in including that.

That said, once we have multiple releases, subsequent releases with confidence of 0 feels repetitive?

Reply:
That's interesting, although a pretty dang subtle way to express that. Is that discoverable?

But it does seem weird to put the same thing on sources, because a `0` at the source level would express "we absolutely do not believe this source at all but put it in", wouldn't it?